tests/basic/pool_function_test.go

```diff
@@ -216,16 +216,47 @@ var _ = Describe("{PoolExpandWithPXRestart}", func() {
 		EndTorpedoTest()
 	})
 
+	It("Restart PX after pool expansion", func() {
+		StartTorpedoTest("RestartAfterPoolExpansion",
+			"Restart PX after pool expansion", nil, testrailID)
```

Comment:
@alicelyu-px I missed this in your PR and assumed that this is an existing test so testrailID would be initialized already. I don't see testrailID being initialized here. Can you help me understand what am I missing? If it is something you missed, please fix it.